README.md

```diff
@@ -81,46 +81,94 @@ Setting the `summarize` argument to `false` lets you check which rows do not mat
   from {{ ref('fct_orders') }}
 {% endset %}
 
-{{ audit_helper.compare_queries(
-    a_query = old_query,
-    b_query = new_query,
-    primary_key = "order_id"
+{{ 
+  audit_helper.compare_and_classify_query_results(
+    old_query, 
+    new_query, 
+    primary_key_columns=['order_id'], 
+    columns=['order_id', 'amount', 'customer_id']
+  )
+}}
+
+```
+
+### compare_and_classify_relation_rows ([source](macros/compare_and_classify_relation_rows.sql))
+
+A wrapper to `compare_which_query_columns_differ`, except it takes two [Relations](https://docs.getdbt.com/reference/dbt-classes#relation) (instead of two queries).
+
+Each relation must have the same columns with the same names, but they do not have to be in the same order.
+
+#### Arguments
+
+- `a_relation` and `b_relation`: The [relations](https://docs.getdbt.com/reference/dbt-classes#relation) you want to compare.
+- `primary_key_columns` (required): A list of primary key column(s) used to join the queries together for comparison.
+- `columns` (optional): The columns present in the two queries you want to compare. Build long lists with a few exclusions with `dbt_utils.get_filtered_columns_in_relation`, or pass `None` and the macro will find all intersecting columns automatically.
+- `sample_limit`: Number of sample records to return per status. Defaults to 20.
+
+#### Usage
+
+```sql
+
+{% set old_relation = adapter.get_relation(
+      database = "old_database",
+      schema = "old_schema",
+      identifier = "fct_orders"
+) -%}
+
+{% set dbt_relation = ref('fct_orders') %}
+
+{{ audit_helper.compare_and_classify_relation_rows(
+    a_relation = old_relation,
+    b_relation = dbt_relation,
+    primary_key_columns = ["order_id"],
+    columns = None
 ) }}
 
 ```
 
-### compare_relations ([source](macros/compare_relations.sql))
+### quick_are_queries_identical ([source](macros/quick_are_queries_identical.sql))
 
-Similar around to `compare_relations`, except it takes two relations (instead of two queries).
+On supported adapters (currently Snowflake and BigQuery), take a hash of all rows in two queries and compare them.
 
-Each relation must have the same columns with the same names, but they do not have to be in the same order. Use `exclude_columns` if some columns only exist in one relation.
+This can be calculated relatively quickly compared to other macros in this package and can efficiently provide reassurance that a refactor introduced no changes.
+
+#### Arguments
```

Comment:
In other macros docs we put arguments after output - total nit pick, but would be nice to be consistent here

Reply:
only because it's you who's asking 😂